lib/datasource/metadata.ts

```diff
@@ -110,11 +110,18 @@ const manualSourceUrls = {
 const githubPages = regEx('^https://([^.]+).github.com/([^/]+)$');
 const gitPrefix = regEx('^git:/?/?');
 
-function massageGithubUrl(url: string): string {
-  return url
+export function massageGithubUrl(url: string): string {
```

Comment:
Can you extract this change to a separate pr please.

Reply:
Sure created https://github.com/renovatebot/renovate/pull/12899